PluralKit.Bot/Commands/Lists/ContextListExt.cs

```diff
@@ -248,4 +248,171 @@ void LongRenderer(EmbedBuilder eb, IEnumerable<ListedMember> page)
             }
         }
     }
+    
+    public static GroupListOptions ParseGroupListOptions(this Context ctx, LookupContext lookupCtx)
```

Comment:
hmm, I wonder if there's any way to do this without duplicating the entire member list code. is there anything that can be shared between MemberListOptions and GroupListOptions?